comment added

ambuj2000 opened the pull request:
I have added a comment

div-bargali commented:
### Pr will be closed and marked as spam. If issue number not found or issue was assigned to someone else.
Marking as spam can block your account from HacktoberFest.

ambuj2000 commented:
I just added a comment and added a file which contains code of inorder,
pre-order and post order....

On Fri, 16 Oct 2020, 19:41 Divyanshu Bargali, <notifications@github.com>
wrote:

> Pr will be closed and marked as spam. If issue number not found or issue
> was assigned to someone else.
>
> Marking as spam can block your account from HacktoberFest.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/div-bargali/Data-Structures-and-Algorithms/pull/345#issuecomment-710072126>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANZGBOWTEQW7VHKNBWCETJDSLBIBTANCNFSM4STMFDGA>
> .
>
